doc/py_tutorials/py_gui/py_drawing_functions/py_drawing_functions.markdown

```diff
@@ -53,7 +53,7 @@ cv.circle(img,(447,63), 63, (0,0,255), -1)
 
 To draw the ellipse, we need to pass several arguments. One argument is the center location (x,y).
 Next argument is axes lengths (major axis length, minor axis length). angle is the angle of rotation
-of ellipse in anti-clockwise direction. startAngle and endAngle denotes the starting and ending of
+of ellipse in clockwise direction. startAngle and endAngle denotes the starting and ending of
```

Comment:
Actually this depends on coordinate system.
With [Cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) it is CCW.
"Display" coordinate system flips Y coordinates direction - confusion comes from here.

Reply:
Thank you, I understood it now.

Reply:
Should I update the doc to mention what you said?